content/learn/other-resources.md

```diff
@@ -39,13 +39,24 @@ IIIF (International Image Interoperability Framework) is an emerging standard fo
 - [IIF Awesome](https://github.com/IIIF/awesome-iiif)
 - [IIIF Apps & Demos](https://iiif.io/apps-demos/#image-servers)
 
+## Large File Storage
```

Comment:
This should go under GitHub rather than be its own category since it's inseparable from that.